BinaryObjectScanner/Packer/WinZipSFX.cs

```diff
@@ -73,20 +73,21 @@ public class WinZipSFX : IExtractableNewExecutable, IExtractablePortableExecutab
         /// <summary>
         /// Handle common extraction between executable types
         /// </summary>
-        private static string? Extract(string file, bool includeDebug)
+        /// <inheritdoc/>
+        public string? Extract(string file, bool includeDebug)
         {
+            if (file == null)
+                return null;
+
 #if NET462_OR_GREATER || NETCOREAPP
             try
             {
-                // Should be using stream instead of file, but stream fails to extract anything. My guess is that the executable portion of the archive is causing stream to fail, but not file.
+                // Create a temp output directory
+                string tempPath = Path.Combine(Path.GetTempPath(), Guid.NewGuid().ToString());
+                Directory.CreateDirectory(tempPath);
+
                 using (ZipArchive zipFile = ZipArchive.Open(file))
                 {
-                    if (!zipFile.IsComplete)
-                        return null;
```

Comment:
Is there a reason that this was removed? Are there incomplete zipfiles that can be extracted?

Reply:
I genuinely don't know, I ported the code from our PKZIP extraction, which doesn't seem to have this check. I figure this means that either this shouldn't have the check either, or the check should be added to our PKZIP extraction.

Reply:
Though, I find it unlikely for there to be incomplete WinZip SFX files like this anyway, as as far as I know, they should always be completely self contained.

Reply:
Please add this back as a minor bit of comfort.